.devcontainer/install-dev-tools.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+
+sudo apt update && sudo apt install -y libtbb-dev git-lfs
```

Comment:
Just curious what we use/need git-lfs for? No problem to install it just wondering where its used 😊